packages/opentelemetry-sdk-trace-base/src/Span.ts

```diff
@@ -202,12 +226,41 @@ export class Span implements api.Span, ReadableSpan {
     this._spanProcessor.onEnd(this);
   }
 
+  private _getTime(inp?: TimeInput): HrTime {
+    if (typeof inp === 'number' && inp < otperformance.now()) {
+      // must be a performance timestamp
+      // apply correction and convert to hrtime
+      return hrTime(inp + this._performanceOffset);
+    }
+
+    if (typeof inp === 'number') {
+      return epochMillisToHrTime(inp);
+    }
+
+    if (inp instanceof Date) {
+      return epochMillisToHrTime(inp.valueOf());
+    }
+
+    if (isTimeInputHrTime(inp)) {
```

Comment:
This version seems to be susceptible to drift in fetch instrumentation (undefined start + hrTime end):

![image](https://user-images.githubusercontent.com/1122555/203392109-6d9916ce-5827-473b-8e65-dc789ce1b836.png)

https://github.com/open-telemetry/opentelemetry-js/blob/3290b25d0a1f28ad0df7efd30ed3ff15dc1c2189/experimental/packages/opentelemetry-instrumentation-fetch/src/fetch.ts#L281

Because hrTime() does `getTimeOrigin() + performance.now()` it would generate drifted HrTime

Reply:
This is because providing a `HrTime` compliant timestamp is passed directly through. My recommendation would be to:

1. Change core.hrTime to return an HrTime generated using `Date.now`
2. Update fetch instrumentation to not pass a time manually

Either of these changes would fix the issue here, but I actually recommend we do both.

Reply:
Alternative to (2) would be to update fetch instrumentation to get start and end times from the performance timing API

Reply:
@t2t2 does the updated version address your concern?

Reply:
fetch and xhr have both been updated. I think this is resolved but I'll wait for @t2t2 to confirm.

Reply:
Yep